services/presentation.js

```diff
@@ -16,10 +16,10 @@ export default {
             /*********connect to microservices*************/
             let returnErr = false;
             let slideServiceRes;
-            let theme = get_sample_theme();
+            //let theme = get_sample_theme();
             rp.get({uri: Microservices.deck.uri + '/deck/' + String(args.id) + '/slides'}).then((res) => {
                 slideServiceRes = JSON.parse(res);
-                callback(null, {content: slideServiceRes.children, theme: theme, selector: selector});
+                callback(null, {content: slideServiceRes.children, theme: slideServiceRes.theme, selector: selector});
```

Comment:
I added this to make use of an update to the /slides route, which now also returns the theme from deck-service